src/app/legacy/containers/PageHandlers/withContexts.jsx

```diff
@@ -57,11 +59,21 @@ const WithContexts = Component => {
             mvtExperiments={mvtExperiments}
             isNextJs={isNextJs}
           >
-            <EventTrackingContextProvider pageData={pageData}>
-              <UserContextProvider>
-                <Component {...props} />
-              </UserContextProvider>
-            </EventTrackingContextProvider>
+            {analytics ? (
+              <EventTrackingContextProvider
+                atiData={{ analytics, title: path(['title'], pageData) }}
+              >
+                <UserContextProvider>
+                  <Component {...props} />
+                </UserContextProvider>
+              </EventTrackingContextProvider>
+            ) : (
+              <EventTrackingContextProvider pageData={pageData}>
+                <UserContextProvider>
+                  <Component {...props} />
+                </UserContextProvider>
+              </EventTrackingContextProvider>
```

Comment:
Let's simplify this, and pass both pageData and atiData as params. 

```suggestion
              <EventTrackingContextProvider pageData={pageData} atiData={atiData}>
                <UserContextProvider>
                  <Component {...props} />
                </UserContextProvider>
              </EventTrackingContextProvider>
```

Reply:
Yep, refactored this with a few other minor changes needed to do so! Much better.